test/test-infra/Makefile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
I think you can delete everythin in test-infra. I don't think you need that directory.